index.html

```diff
@@ -0,0 +1,194 @@
+        /></a>
+        <ul>
+          <li><a href="./index.html">Home</a></li>
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#">Job Offer</a></li>
+          <li><a href="#">Help</a></li>
+          <li class="login">
+            <a href="#"><span>BUY YOUR </span>CAR NOW</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="paragraphe">
+        <div class="paragraphe_title">
+          <h3>Welcome to your best car's sale company</h3>
+          <h1>Hyundai Rwanda. Sales and Marketting Place</h1>
+
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Nemo
+            doloremque tempora voluptas maxime? Nulla ex, magni odit nihil
+            aspernatur itaque fugiat numquam odio veritatis placeat perferendis,
+            enim suscipit quas ipsa blanditiis! Et consequatur reiciendis
+            temporibus, animi earum ipsa id distinctio dolor tempora mollitia,
+            similique?
+          </p>
+          <div class="span_para">
+            <h4>07 FEB 2022(Monday) - 16(Wednesday) CAR SOLD</h4>
+          </div>
+          <small
+            >@National center of car production, sales and transportation</small
+          >
+        </div>
+      </section>
+
+      <section id="courses" class="my-team">
+        <p>Executive Team</p>
+        <hr />
+        <ul class="team">
+          <li class="team-list">
+            <img src="./images/user1.jpg" alt="" class="flex1" />
+            <p class="title">CEO</p>
+            <p class="flex3">
+              Executive founder of Hyundai based in North-Korea
+            </p>
+          </li>
+          <li>
+            <img src="./images//seccretaire.jpg" alt="" class="flex1" />
+            <p class="title">Advisor</p>
+            <p class="flex3">
+              Magistral Advisor of the company <br />
+              molestiae inventore deserunt?
+            </p>
+          </li>
+          <li>
+            <img src="./images/user2.jpg" alt="" class="flex1" />
+            <p class="title">COO</p>
+            <p class="flex3">
+              Co-founder of Hyundai Brand based in Ethiopia<br />
+              molestiae inventore deserunt?
+            </p>
+          </li>
+          <li>
+            <img
+              src="./images/pexels-orione-conceição-7527617.jpg"
+              alt=""
+              class="flex1"
+            />
+            <p class="title">Marketing</p>
+            <p class="flex3">
+              Agen marketing specialist with PHD<br />
+              molestiae inventore deserunt?
+            </p>
+          </li>
+          <li>
+            <img src="./images/msrket.jpg" alt="" class="flex1" />
+            <p class="title">Secretary</p>
+            <p class="flex3">
+              Executive secretary of the Brand<br />
+              molestiae inventore deserunt?
+            </p>
+          </li>
+        </ul>
```

Comment:
- [ ] Kindly use HTML semantic `article` instead of `ul`. Using semantic tags is considered one of the best practices in the programming world.